salesagility/SuiteCRM
# Noon theme

## Description




Cherry-picked and squashed #8540 onto latest develop.

## How To Test This

1. Select noon theme in user profile.
2. Test functionality for all 5 themes and make sure dropdowns, edit views, detail views, etc look the way they are supposed to.

## Types of changes


### Final checklist
